### PR TITLE
docs: Replaced deprecated <Set private> with PrivateSet within router.md

### PR DESCRIPTION
**Pull request by suzdalnitski**

### CODE REVIEW

File: docs/docs/router.md [hunk 1]
```diff
@@ -572,7 +559,7 @@ When the lazy-loaded page is loading, `PageLoadingContext.Consumer` will pass `{
 
 Let's say you have a dashboard area on your Redwood app, which can only be accessed after logging in. When Redwood Router renders your private page, it will first fetch the user's details, and only render the page if it determines the user is indeed logged in.
 
-In order to display a loader while auth details are being retrieved you can add the `whileLoadingAuth` prop to your private `<Route>`, `<Set private>` or the `<PrivateSet>` component:
+In order to display a loader while auth details are being retrieved you can add the `whileLoadingAuth` prop to your private `<Route>`, or the `<PrivateSet>` component:
```
Comment (jtoar) [inline docs/docs/router.md:562]:
```suggestion
In order to display a loader while auth details are being retrieved you can add the `whileLoadingAuth` prop to your private `<Route>` or `<PrivateSet>` component:
```

File: docs/docs/router.md [hunk 2]
```diff
@@ -128,31 +128,18 @@ becomes...
 </MainLayout>
 ```
 
-### `private` Set
+### `PrivateSet`
 
-Sets can take a `private` prop which makes all Routes inside that Set require authentication. When a user isn't authenticated and attempts to visit one of the Routes in the private Set, they'll be redirected to the Route passed as the Set's `unauthenticated` prop. The originally-requested Route's path is added to the query string as a `redirectTo` param. This lets you send the user to the page they originally requested once they're logged-in.
+A `PrivateSet` makes all Routes inside that Set require authentication. When a user isn't authenticated and attempts to visit one of the Routes in the PrivateSet, they'll be redirected to the Route passed as the PrivateSet's `unauthenticated` prop. The originally-requested Route's path is added to the query string as a `redirectTo` param. This lets you send the user to the page they originally requested once they're logged-in.
```
Comment (jtoar) [inline docs/docs/router.md:133]:
```suggestion
A `PrivateSet` makes all Routes inside that Set require authentication. When a user isn't authenticated and attempts to visit one of the Routes in the `PrivateSet`, they'll be redirected to the Route passed as the `PrivateSet`'s `unauthenticated` prop. The originally-requested Route's path is added to the query string as a `redirectTo` param. This lets you send the user to the page they originally requested once they're logged-in.
```

**Review by jtoar — COMMENTED** [review]
Thanks @suzdalnitski! I can't remember if @Tobbe wanted to leave in references to Private Set till they were removed so I'll ping him for a second pair of eyes on this one. For now just added a few minor comments

**Comment by Tobbe** [comment]
> I can't remember if @Tobbe wanted to leave in references to Private Set till they were removed

Nah, for the docs I think we can remove them all 👍

**Pull request merged**